### PR TITLE
Tabulated benchmark result

### PR DESCRIPTION
**Pull request by babenek**
- Skip install unused packages
- meta is loaded once in list to reduce disc reading
- refactoring report output in table view
- refactoring and optimisation
based on #41 

new report looks like:
```
DATA: 19434458 valid lines. MARKUP: 73692 items
Category                      Positives    Negatives    Template
--------------------------  -----------  -----------  ----------
Authentication Key & Token           67            1          31
Generic Secret                     1055           15         203
Generic Token                       333           45         558
Other                               511        63135         635
Password                           1403          110        4170
Predefined Pattern                  326            2          40
Private Key                        1001            1           3
Seed, Salt, Nonce                    39            4           4
TOTAL:                             4735        63313        5644
Detected Credentials: 4721
credsweeper result_cnt : 4208, lost_cnt : 0, true_cnt : 3844, false_cnt : 233
Category                      TP    FP        TN    FN         FPR        FNR       ACC       PRC       RCL        F1
--------------------------  ----  ----  --------  ----  ----------  ---------  --------  --------  --------  --------
Authentication Key & Token    51     0        32    16              0.238806   0.838384  1         0.761194  0.864407
Generic Secret               971     1       217    84  0.00458716  0.0796209  0.933229  0.998971  0.920379  0.958066
Generic Token                284     1       602    49  0.00165837  0.147147   0.946581  0.996491  0.852853  0.919094
Other                        246   226     63544   265  0.00354399  0.518591   0.992362  0.521186  0.481409  0.500509
Password                     981     2      4278   422  0.00046729  0.300784   0.925392  0.997965  0.699216  0.822297
Predefined Pattern           309     1        41    17  0.0238095   0.0521472  0.951087  0.996774  0.947853  0.971698
Private Key                  967     0         4    34              0.033966   0.966169  1         0.966034  0.982724
Seed, Salt, Nonce             35     2         6     4  0.25        0.102564   0.87234   0.945946  0.897436  0.921053
                            3844   233  19429490   891  1.199e-05   0.188173   0.999942  0.94285   0.811827  0.872447


```

### CODE REVIEW

File: meta/3d82970b.csv [hunk 1]
```diff
@@ -27,6 +27,7 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 34073,4ea0570b,GitHub,3d82970b,data/3d82970b/test/i.js,5:5,T,F,13,97,F,F,Any,,JSON Web Token,Token,5.10,84,F,F,F,Predefined Pattern
 34074,39a14b64,GitHub,3d82970b,data/3d82970b/test/k.js,13:13,T,F,13,134,F,F,Any,,JSON Web Token,Token,5.33,121,F,F,F,Predefined Pattern
 34483,d6336c49,GitHub,3d82970b,data/3d82970b/test/j.js,224:224,T,F,13,157,F,F,Any,,JSON Web Token,Token,5.13,144,F,F,F,Predefined Pattern
+
```
Comment (babenek) [inline meta/3d82970b.csv:30]:
```suggestion
```

File: benchmark/scanner/scanner.py [hunk 2]
```diff
@@ -17,6 +20,66 @@ def __init__(self, scanner_type: str, scanner_url: str, working_dir: str, cred_d
         self.true_cnt: int = 0
         self.false_cnt: int = 0
         self.result_dict: dict = {}
+        self.total_true_cnt = 0
+        self.total_false_cnt = 0
+        self.total_template_cnt = 0
+        self.categories: Dict[str, Tuple[int, int]] = {}  # true_cnt, false_cnt
```
Comment (babenek) [inline benchmark/scanner/scanner.py:26]:
```suggestion
        self.categories: Dict[str, Tuple[int, int, int]] = {}  # true_cnt, false_cnt, template_cnt
```

File: benchmark/scanner/scanner.py [hunk 3]
```diff
@@ -17,6 +20,66 @@ def __init__(self, scanner_type: str, scanner_url: str, working_dir: str, cred_d
...
+                            self.total_true_cnt += 1
+                        elif row["GroundTruth"] == "F":
+                            self.total_false_cnt += 1
+                            false_cnt += 1
+                        elif row["GroundTruth"] == "Template":
+                            self.total_template_cnt += 1
+                            template_cnt += 1
+                        else:
+                            # wrong markup should be detected
+                            print(row, flush=True)
```
Comment (csh519) [inline benchmark/scanner/scanner.py:56]: What about add some string which make recognize the row is marked up wrong more easily?

```suggestion
                            # wrong markup should be detected
                            print(f"[WRONG MARKUP] {row}", flush=True)
```

**Review by csh519 — APPROVED** [review]
Thank you for adding `tabluate` module to beautify the results.

LGTM :+1:

**Review by kmnls — APPROVED** [review]

**Pull request merged**